hvpy/facade.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+from hvpy.core import execute_api_call
+from hvpy.parameters import getJP2ImageInputParameters
+
+__all__ = ["getJP2Image"]
+
+
+def copy_docstring(input_class):
+    def decorator(func):
+        func.__doc__ = func.__doc__ + input_class.__doc__
```

Comment:
I am currently, appending the `class docstring` to the facade `function docstring`.

result is:
```
    Get a JP2 image from the helioviewer.org API.
    
    Handles the input parameters of the getJP2Image API.

    Attributes
    ----------

...etc
```

Reply:
You will want to remove the first line of the `input_class` documentation string.